catalog/spatial/registry/registry-report-viewer/src/main/resources/templates/report.hbt

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type" />
+    <!-- styles -->
+    <link href="/services/registries/assets/css/bootstrap.min.css" rel="stylesheet" />
+    <style type="text/css">
+        body {
+        background-color: {
+        {
+        background
+        }
+        }
+        ;
+        color: {
+        {
+        foreground
+        }
+        }
+        ;
+        margin-left: 15px;
+        }
+
+        .scrollable-list {
+        height: auto;
+        max-height: 600px;
+        overflow: auto;
+        }
+
+        .spinner {
+        height: 100px;
+        position: relative;
+        }
+
+        .banner-container {
+        text-align: center;
+        height: 20px;
+        }
+
+        .header-layout,
+        .footer-layout,
+        #content {
+        position: fixed;
+        right: 0;
+        left: 0;
+        }
+
+        .header-layout {
+        top: 0;
+        }
+
+        .footer-layout {
+        bottom: 0;
+        }
+
+        #content {
+        top: 0px;
+        bottom: 0px;
+        overflow: auto;
+        }
+
+        .jumbotron {
+        background-color: rgba(0, 0, 0, 0.25);
+        }
+
+        .panel {
+        color: black;
+        }
+
+        .table {
+        margin-left: 50px;
+        border: 1px solid #ddd;
+        max-width: 2000px;
+        }
+
+        h3 {
+        margin-left: 25px
+        }
+
+        h4 {
+        margin-left: 50px
+        }
+    </style>
+    <link href="/services/registries/assets/css/bootstrap-theme.min.css" rel="stylesheet" />
+</head>
+
+<body>
+<h1>{{Name}}</h1>
+<table class="table table-striped">
+    <tbody>
+    {{#each General}}
+    <tr>
+        <td width="250 px">
+            <strong>{{@key}}</strong>
```

Comment:
But does the key describe the value adjacent to it? If it does, it's kind of like a header. Either way works for me.